.github/ci-hpc-config.yml

```diff
@@ -0,0 +1,18 @@
+build:
+  modules:
```

Comment:
Probably naive question, but the PR i reviewed for anemoi-models yesterday, in this file it was listed python: '3.10' , we don't we need that here?